### PR TITLE
DOCSP-31825: Update titles

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-31825>
Staging - <https://docs-mongodbcom-staging.corp.mongodb.com/golang/docsworker-xlarge/DOCSP-31825-update-titles/fundamentals/crud/write-operations/modify/>
<https://docs-mongodbcom-staging.corp.mongodb.com/golang/docsworker-xlarge/DOCSP-31825-update-titles/fundamentals/crud/write-operations/delete/>

Redirects - 
Redirect 301 /docs/drivers/go/1.7/fundamentals/crud/write-operations/change-a-document/ https://www.mongodb.com/docs/drivers/go/1.7/fundamentals/crud/write-operations/modify/
Redirect 301 /docs/drivers/go/1.8/fundamentals/crud/write-operations/change-a-document/ https://www.mongodb.com/docs/drivers/go/1.8/fundamentals/crud/write-operations/modify/
Redirect 301 /docs/drivers/go/1.9/fundamentals/crud/write-operations/change-a-document/ https://www.mongodb.com/docs/drivers/go/1.9/fundamentals/crud/write-operations/modify/
Redirect 301 /docs/drivers/go/1.10/fundamentals/crud/write-operations/change-a-document/ https://www.mongodb.com/docs/drivers/go/1.10/fundamentals/crud/write-operations/modify/
Redirect 301 /docs/drivers/go/1.11/fundamentals/crud/write-operations/change-a-document/ https://www.mongodb.com/docs/drivers/go/1.11/fundamentals/crud/write-operations/modify/
Redirect 301 /docs/drivers/go/1.12/fundamentals/crud/write-operations/change-a-document/ https://www.mongodb.com/docs/drivers/go/1.12/fundamentals/crud/write-operations/modify/
Redirect 301 /docs/drivers/go/master/fundamentals/crud/write-operations/change-a-document/ https://www.mongodb.com/docs/drivers/go/master/fundamentals/crud/write-operations/modify/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [ ] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [x] Are all the links working?

### CODE REVIEW

File: config/redirects [hunk 1]
```diff
@@ -10,4 +10,5 @@ raw: ${prefix}/stable -> ${base}/current/
 [*-master]: ${prefix}/${version}/fundamentals/crud/read-operations/watch/ -> ${base}/${version}/fundamentals/crud/read-operations/changestream/
 [*-master]: ${prefix}/${version}/usage-examples/watch/ -> ${base}/${version}/usage-examples/changestream/
 [*-master]: ${prefix}/${version}/fundamentals/crud/run-command/ -> ${base}/${version}/fundamentals/run-command/
-[*-v1.11]: ${prefix}/${version}/fundamentals/logging/ -> ${base}/${version}/
\ No newline at end of file
+[*-v1.11]: ${prefix}/${version}/fundamentals/logging/ -> ${base}/${version}/
+[*-master]: ${prefix}/${version}/fundamentals/crud/write-operations/change-a-document/ -> ${base}/${version}/fundamentals/crud/write-operations/modify/
```
Comment (rustagir) [inline config/redirects:14]: tip: when you add a redirect, paste the output of the `mut-redirects` test in the description of the PR so the reviewer can do a double check!

**Review by rustagir — APPROVED** [review]
approving with a few small suggestions!

File: source/fundamentals/crud/write-operations/delete.txt [hunk 2]
```diff
@@ -1,7 +1,7 @@
 .. _golang-delete-guide:
 
 =================
-Delete a Document
+Delete Documents
 =================
 
 .. default-domain:: mongodb
```
[the remark this answers is not in the document]
Comment (norareidy) [reply source/fundamentals/crud/write-operations/delete.txt]: Good to know

File: config/redirects (hunk 1 above)
Comment (norareidy) [reply config/redirects:14]: Gotcha, added that in (just the output for the new redirect)

**Pull request merged**